src/components/claims/claim-details/components/MemberInformation.tsx

```diff
@@ -56,9 +62,13 @@ const MemberInformation: React.SFC<{ member: Member }> = ({ member }) => {
       <p>
         <b>Personal Number:</b> {member.personalNumber}
       </p>
-      <p>
-        <b>Address:</b> {member.address}, {member.postalNumber} {member.city}
-      </p>
+      {address && (
+        <p>
+          <b>Address:</b> {address?.street}, {address?.postalCode}{' '}
+          {address.city}
+        </p>
+      )}
```

Comment:
```suggestion
      {address && (
        <p>
          <b>Address:</b> {address.street}, {address.postalCode}{' '}
          {address?.city}
        </p>
      )}
```
Since the schema is `type Address {
  street: String!
  postalCode: String!
  city: String
}` and we're doing null check with `address &&` 🤪